views/docs/index.hbs

```diff
@@ -7,108 +7,109 @@
-            <div class="docs-guides-readme">
-              {{{detailedTutorials.html}}}
-            </div>
+        <div class="col-ms-12 col-lg-3 mx-3">
+          <h2 class="docs-heading pb-3 mb-3 docs-title">
+            <span class="mega-octicon octicon-tools pr-3"></span>{{localized.docs.detailed}}
+          </h2>
+          <div class="docs-guides-readme">
+            {{{detailedTutorials.html}}}
           </div>
+        </div>
 
-          <div class="col-ms-12 col-lg-3 mx-3">
-            <!-- Quick Glossary -->
-            <h2 class="docs-heading pb-3 mb-3"><a class="docs-title" href="/docs/api"><span class="mega-octicon octicon-gear pr-3"></span>{{localized.docs.api_reference}}</a></h2>
-            <div class="docs-guides-readme">
-              {{{apiReference.html}}}
-            </div>
-
-            <!-- Custom DOM Elements -->
-            <h4 class="docs-sub-heading pb-3 mb-3">
-              {{localized.docs.custom_dom}}
-            </h4>
-            <div class="docs-guides-readme">
-              {{{customDomElements.html}}}
-            </div>
+        <div class="col-ms-12 col-lg-3 mx-3">
+          <!-- Quick Glossary -->
+          <h2 class="docs-heading pb-3 mb-3"><a class="docs-title" href="/docs/api"><span class="mega-octicon octicon-gear pr-3"></span>{{localized.docs.api_reference}}</a></h2>
+          <div class="docs-guides-readme">
+            {{{apiReference.html}}}
+          </div>
 
-            <!-- Main Process Modules -->
-            <h4 class="docs-sub-heading pb-3 mb-3">
-              {{localized.docs.main_proc_modules}}
-            </h4>
-            <div class="docs-guides-readme">
-              {{{mainProcModules.html}}}
-            </div>
+          <!-- Custom DOM Elements -->
+          <h3 class="docs-sub-heading pb-3 mb-3">
+            {{localized.docs.custom_dom}}
+          </h3>
+          <div class="docs-guides-readme">
+            {{{customDomElements.html}}}
+          </div>
 
-            <!-- Renderer Process Modules -->
-            <h4 class="docs-sub-heading pb-3 mb-3">
-              {{localized.docs.renderer_proc_modules}}
-            </h4>
-            <div class="docs-guides-readme">
-              {{{rendererProcModules.html}}}
-            </div>
+          <!-- Main Process Modules -->
+          <h3 class="docs-sub-heading pb-3 mb-3">
+            {{localized.docs.main_proc_modules}}
+          </h3>
+          <div class="docs-guides-readme">
+            {{{mainProcModules.html}}}
+          </div>
 
-            <!-- Both Process Types Modules -->
-            <h4 class="docs-sub-heading pb-3 mb-3">
-              {{localized.docs.both_proc_modules}}
-            </h4>
-            <div class="docs-guides-readme">
-              {{{bothProcModules.html}}}
-            </div>
+          <!-- Renderer Process Modules -->
+          <h4 class="docs-sub-heading pb-3 mb-3">
+            {{localized.docs.renderer_proc_modules}}
+          </h4>
```

Comment:
Should this be an h3 to match the "main process modules" & the "both process types modules" headers?
```suggestion
          <h3 class="docs-sub-heading pb-3 mb-3">
            {{localized.docs.renderer_proc_modules}}
          </h3>
```

Reply:
Oh yeah, good catch. I originally changed those to `h3` for the whole to be more semantic (i.e. not skipping a level of headings), but this one just slipped through.